projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-documents/documenten-api-documents.component.html

```diff
@@ -80,6 +82,35 @@
   }
 
   <valtimo-documenten-api-metadata-modal
+    [author]="obs.formFields?.auteur?.defaultValue"
+    [confidentialityLevel]="obs.formFields?.vertrouwelijkheidaanduiding?.defaultValue"
+    [description]="obs.formFields?.beschrijving?.defaultValue"
+    [disableAuthor]="obs.formFields?.auteur?.readonly"
+    [disableConfidentialityLevel]="obs.formFields?.vertrouwelijkheidaanduiding?.readonly"
+    [disableCreationDate]="obs.formFields?.creatiedatum?.readonly"
+    [disableDescription]="obs.formFields?.beschrijving?.readonly"
+    [disableDocumentTitle]="obs.formFields?.titel?.readonly"
+    [disableDocumentType]="obs.formFields?.informatieobjecttype?.readonly"
+    [disableFilename]="obs.formFields?.bestandsnaam?.readonly"
+    [disableLanguage]="obs.formFields?.taal?.readonly"
+    [disableStatus]="obs.formFields?.status?.readonly"
+    [documentTitle]="obs.formFields?.titel?.defaultValue"
+    [documentType]="obs.formFields?.informatieobjecttype?.defaultValue"
+    [filename]="obs.formFields?.bestandsnaam?.defaultValue"
+    [hideAdditionalDate]="!obs.formFields?.aanvullendeDatum?.visible"
+    [hideAuthor]="!obs.formFields?.auteur?.visible"
+    [hideConfidentialityLevel]="!obs.formFields?.vertrouwelijkheidaanduiding?.visible"
+    [hideCreationDate]="!obs.formFields?.creatiedatum?.visible"
+    [hideDescription]="!obs.formFields?.beschrijving?.visible"
+    [hideDocumentTitle]="!obs.formFields?.titel?.visible"
+    [hideDocumentType]="!obs.formFields?.informatieobjecttype?.visible"
+    [hideFilename]="!obs.formFields?.bestandsnaam?.visible"
+    [hideLanguage]="!obs.formFields?.taal?.visible"
+    [hideStatus]="!obs.formFields?.status?.visible"
+    [hideTags]="!obs.formFields?.trefwoorden?.visible"
+    [language]="obs.formFields?.taal?.defaultValue"
+    [status]="obs.formFields?.status?.defaultValue"
+    [tags]="obs.metadata?.trefwoorden"
```

Comment:
This is way too many Inputs, convert them into two different objects: config and prefillData. Config has all the 'hide' inputs and prefillData has the prefilled Data

Reply:
I tried to reduce it a bit